qiskit_experiments/quantum_volume/qv_experiment.py

```diff
@@ -0,0 +1,249 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Quantum Volume Experiment class.
+"""
+
+import copy
+from typing import Union, Iterable, Optional, List
+from numpy.random import Generator, default_rng
+from qiskit.providers.backend import Backend
+from qiskit.providers.options import Options
+
+try:
+    from qiskit import Aer
+
+    HAS_SIMULATION_BACKEND = True
+except ImportError:
+    HAS_SIMULATION_BACKEND = False
+
+from qiskit import QuantumCircuit
+from qiskit.circuit.library import QuantumVolume
+from qiskit import transpile, assemble
+from qiskit import execute
+from qiskit.exceptions import QiskitError
+from qiskit.providers.basebackend import BaseBackend as LegacyBackend
+from qiskit_experiments.base_experiment import BaseExperiment
+from qiskit_experiments.experiment_data import ExperimentData
+from .qv_analysis import QVAnalysis
+
+
+class QVExperiment(BaseExperiment):
+    """Quantum Volume Experiment class
+
+    Experiment Options:
+        trials (int): Optional, number of times to generate new Quantum Volume circuits and
+                    calculate their heavy output.
+    """
+
+    # Analysis class for experiment
+    __analysis_class__ = QVAnalysis
+
+    # ExperimentData class for the simulations
+    __simulation_data__ = ExperimentData
+
+    def __init__(
+        self,
+        qubits: Union[int, Iterable[int]],
+        trials: Optional[int] = 1,
```

Comment:
```suggestion
        trials: Optional[int] = 100,
```
Since this experiment raises a warning if trials < 100, we should make the default value something reasonable (ie >=100).